src/integrationTest/resources/application.yaml

```diff
@@ -1,7 +1,18 @@
 spring:
   application.name: Hello
   flyway.locations: classpath:db/migration
-
+  main:
+    allow-bean-definition-overriding: true
+  security:
+    oauth2:
+      client:
+        provider:
+          oidc:
+            issuer-uri: ${OPEN_ID_API_BASE_URI:https://idam-web-public.aat.platform.hmcts.net/o}
+        registration:
```

Comment:
everywhere else issuer url is pointing to forge rock but here in  integration test yaml file, you have given the url with different host name.. may not be an issue but i just wanted to check if it is not an versight

Reply:
oidc.iIssuer-uri  and oidc.issuer  are both different and everywhere pointing correct only .